cli/cmd/generate.go

```diff
@@ -207,7 +207,12 @@ func writeHclOutputPreCheck(outputLocation string) (bool, error) {
 		return false, err
 	}
 
-	hclPath := filepath.FromSlash(fmt.Sprintf("%s/main.tf", outputDir))
+	fileName := cloud
```

Comment:
Any need for reassignment here. Maybe filename could be used as the arg name instead of cloud.

Reply:
fair point. Will update